### PR TITLE
ras/lparnumascore: Update supported distro release check

### PR DESCRIPTION
**Pull request by sacsant**
lparnumascore command is available for SuSE distro releases.
Remove the code restriction accordingly

Also fix code style warning.
    
Signed-off-by: Sachin Sant <sachinp@linux.ibm.com>

### CODE REVIEW

**Comment by sacsant** [comment]
Before the fix:

avocado run --test-runner runner ras/lparnumascore.py
JOB ID     : 8671f6444fd7a570eee32f655c24a5685b498f01
JOB LOG    : /home/results/job-2022-04-15T09.16-8671f64/job.log
 (1/1) ras/lparnumascore.py:lparnumascore.test_lparnumascore: CANCEL: lparnumascore is not supported on SuSE (0.01 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 1
JOB HTML   : /home/results/job-2022-04-15T09.16-8671f64/results.html
JOB TIME   : 8.51 s


After the fix:

avocado run --test-runner runner ras/lparnumascore.py
JOB ID     : c27497d1ec0ec99aebdde058f5210e697e1c35bd
JOB LOG    : /home/results/job-2022-04-15T09.23-c27497d/job.log
 (1/1) ras/lparnumascore.py:lparnumascore.test_lparnumascore: PASS (0.05 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/results/job-2022-04-15T09.23-c27497d/results.html
JOB TIME   : 8.31 s

**Review by PraveenPenguin — APPROVED** [review]

File: ras/lparnumascore.py [hunk 1]
```diff
@@ -27,22 +27,22 @@ class lparnumascore(Test):
...
+                        shell=True)).exit_status:
             self.is_fail += 1
         return
 
     @skipIf(IS_POWER_NV, "This test is supported on PowerVM environment")
     def setUp(self):
         det_dist = distro.detect()
-        if det_dist.name == 'SuSE' or det_dist.name \
-           == 'rhel' and int(det_dist.version) <= 8:
+        if det_dist.name == 'rhel' and int(det_dist.version) <= 8:
```
Comment (Naresh-ibm) [inline ras/lparnumascore.py:45]: @sacsant So this support from RHEL9 on wards ?

Comment (sacsant) [reply ras/lparnumascore.py:45]: RHEL 8.x onwards as well

Comment (sacsant) [reply ras/lparnumascore.py:45]: ah I see the check for RHEL is wrong .. it should be >= . I will update the patch to fix this as well. Good catch.

**Comment by Naresh-ibm** [comment]
LGTM

**Review by Naresh-ibm — APPROVED** [review]

**Comment by sacsant** [comment]
Results from RHEL 8.x

avocado run --test-runner runner ras/lparnumascore.py
JOB ID     : 6a4373f8aa365d7dc9ac8d24d0f89aae4f057489
JOB LOG    : /home/avocado/results/job-2022-04-18T07.19-6a4373f/job.log
 (1/1) ras/lparnumascore.py:lparnumascore.test_lparnumascore: PASS (1.01 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado/results/job-2022-04-18T07.19-6a4373f/results.html
JOB TIME   : 1.52 s

**Pull request merged**